hypermap/aggregator/tasks.py

```diff
@@ -81,6 +90,43 @@ def check_layer(self, layer):
         task_error.save()
 
 
+@shared_task(bind=True)
+def index_cached_layers(self):
+    """
+    Index all layers in the Django cache (Index all layers who have been checked).
+    """
+    from hypermap.aggregator.models import Layer
+    from hypermap.aggregator.solr import SolrHypermap
+    from hypermap.aggregator.models import TaskError
```

Comment:
Why do you have inline imports? isn't it better to do it at the start of the module to make sure there are no circular imports?